dev-aws/kafka-shared-msk/iam/iam.tf

```diff
@@ -285,3 +285,10 @@ module "iam_credentials_public_events_mapper" {
   consume_topics   = [(kafka_topic.iam_credentials_v1.name)]
   consume_groups   = ["iam.public-events-mapper-iam-credentials_v1"]
 }
+
+module "cbc_fraud_detection_consumer" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = [kafka_topic.iam_credentials_v1_public.name]
+  consume_groups   = ["cbc.cbc-fraud-detection-consumer-v1"]
```

Comment:
I just realised that this is the same app as declared in cbc. I think we should skip declaring the consumer group here, as this will reference the same ACL resource for the same consumer group defined in CBC, resulting in managing the same resource from 2 different tf modules.

Reply:
fixed!